modules/cluster-management/src/test/java/org/apache/ignite/internal/cluster/management/raft/AbstractClusterStateStorageManagerTest.java

```diff
@@ -213,4 +216,22 @@ void testValidatedNodes() {
 
         assertThat(storageManager.getValidatedNodeIds(), containsInAnyOrder("node2"));
     }
+
+    private static ClusterTag clusterTag(String cluster) {
```

Comment:
Can these methods be taken out in a separate auxiliary class?

Reply:
Sure